docs/linking.md

```diff
@@ -11,6 +11,9 @@ The `<g-link>` component is available globally in all your Pages, Templates & Co
 ### Link between pages
 To link to `~/src/pages/About.vue` use `<g-link to="/about" />`.
 
+### Link from parent to children pages
+To link from `~/src/pages/about/Index.vue` to `~/src/pages/about/Careers.vue` use `<g-link :to="'careers'" append />`.
```

Comment:
It's a bit confusing to call `Index.vue` and `Careers.vue` parent and child pages because, in terms of file system hierarchy, the source files are siblings. Could we change it to like `~/src/pages/about/careers/Index.vue` so the parent-child relationship is unambiguous?

I think the `to` can be simplified to just `to="careers"`. If the `append` attribute requires a `v-bind` on the `to`, we should explain that.

(optional): Would it be possible to document `append` in the table at the bottom?